sandbox-backend/src/main/java/com/exadel/sandbox/team5/util/RateSetter.java

```diff
@@ -0,0 +1,13 @@
+package com.exadel.sandbox.team5.util;
+
+import lombok.*;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+public class RateSetter {
```

Comment:
`EntityRateDto` or `DiscountRateDto` will be better

Reply:
done